content/ja/docs/tutorials/stateful-application/mysql-wordpress-persistent-volume.md

```diff
@@ -234,7 +234,7 @@ WordPressのインストールをこのページのまま放置してはいけ
 ## {{% heading "whatsnext" %}}
 
 
-* [イントロスペクションとデバッグ](/docs/tasks/debug/debug-application)についてさらに学ぶ
+* [イントロスペクションとデバッグ](/ja/docs/tasks/debug/debug-application)についてさらに学ぶ
```

Comment:
```suggestion
* [イントロスペクションとデバッグ](/ja/docs/tasks/debug/debug-application/debug-running-pod/)についてさらに学ぶ
```
Suggestion to match this link to the original.
https://github.com/kubernetes/website/blob/38418fbc9f11524a832aa2f1fd03f696e7f5594a/content/en/docs/tutorials/stateful-application/mysql-wordpress-persistent-volume.md?plain=1#L239

This fix can also be categorized as "Change path and add ja" in the issue https://github.com/kubernetes/website/issues/38797, but can fix it in this PR.
Because this is a special case unrelated to directory structure changes about debug pages.

Reply:
Thanks for your review! I fixed the path by adding `/debug-running-pod/`. Please check it.

https://github.com/kubernetes/website/pull/39020/commits/e5a53291b54a7e1cd847a77c9133c90b39ecf27e